app/models/export/ProductDraftWrapper.java

```diff
@@ -0,0 +1,266 @@
+                .description(getDescription())
+                .taxCategory(taxCategoryReference)
+                .variants(Arrays.asList(mediumBox(), largeBox(), hugeBox()))
+                .build();
+    }
+
+    private ProductVariantDraft smallBox() {
+        return createVariant(productVariantDraftWrapper -> "box6".equals(productVariantDraftWrapper.getSku()));
+    }
+
+    private ProductVariantDraft mediumBox() {
+        return createVariant(productVariantDraftWrapper -> "box12".equals(productVariantDraftWrapper.getSku()));
+    }
+
+    private ProductVariantDraft largeBox() {
+        return createVariant(productVariantDraftWrapper -> "box24".equals(productVariantDraftWrapper.getSku()));
+    }
+
+    private ProductVariantDraft hugeBox() {
+        return createVariant(productVariantDraftWrapper -> "box36".equals(productVariantDraftWrapper.getSku()));
+    }
+
+    private ProductVariantDraft createVariant(final Predicate<ProductVariantDraftWrapper> boxPredicate) {
+        final ProductVariantDraftWrapper variantWrapper = getVariants().stream()
+                .filter(boxPredicate).findFirst()
+                .orElseThrow(() -> new RuntimeException("Unable to find variant"));
+
+        final List<PriceWrapper> pricesWrappers = variantWrapper.getPrices();
+        final List<Price> prices = pricesWrappers.stream().map(priceWrapper ->
+                PriceBuilder.of(Money.of(priceWrapper.getValue().getAmount(), priceWrapper.getValue().getCurrencyCode()))
+                        .build()).collect(Collectors.toList());
+
+        final List<ImageWrapper> imageWrappers = variantWrapper.getImages();
+        final List<Image> images = imageWrappers.stream().map(imageWrapper ->
+                Image.ofWidthAndHeight(imageWrapper.url, imageWrapper.getDimensions().getWidth(),
+                        imageWrapper.getDimensions().getHeight(),
+                        "donuts image label")).collect(Collectors.toList());
+
+        final List<AttributeDraftWrapper> attributeDraftWrappers = variantWrapper.getAttributes();
+        final List<AttributeDraft> attributes = attributeDraftWrappers.stream()
+                .map(attributeDraftWrapper -> AttributeDraft.of(attributeDraftWrapper.getName(),
+                        attributeDraftWrapper.getValue()))
+                .collect(Collectors.toList());
+
+        return ProductVariantDraftBuilder.of().prices(prices).images(images).sku(variantWrapper.getSku())
+                .attributes(attributes).build();
+    }
+
+    public LocalizedString getDescription() {
+        return description;
+    }
+
+    public LocalizedString getName() {
+        return name;
+    }
+
+    public LocalizedString getSlug() {
+        return slug;
+    }
+
+    public ProductVariantDraftWrapper getMasterVariant() {
+        return masterVariant;
+    }
+
+    public List<ProductVariantDraftWrapper> getVariants() {
+        return variants;
+    }
+
+    public static class ProductVariantDraftWrapper {
```

Comment:
I would prefer this as normal class for SRP. https://en.wikipedia.org/wiki/Single_responsibility_principle